tests/e2e_tests/Dockerfile.gui

```diff
@@ -0,0 +1,23 @@
+FROM python:2
+WORKDIR /usr/src/app
+ENV CHROME_BIN="/usr/bin/google-chrome"
+ENV DISPLAY=":99.0"
+RUN apt-get -q update && apt-get -yq install chromium=73.0.3683.75-1~deb9u1
+RUN wget -q "https://chromedriver.storage.googleapis.com/73.0.3683.68/chromedriver_linux64.zip" -O /tmp/chromedriver.zip \
+  && unzip /tmp/chromedriver.zip -d /usr/bin/ \
+  && rm /tmp/chromedriver.zip
+RUN pip install -U --user paramiko 'fabric<2.0' PyYAML pytest requests pytest-xdist filelock selenium psutil
+RUN ln -s /usr/bin/chromedriver .
+COPY tests .
+RUN mkdir -p downloaded-tools && \
+  curl -SL --fail "https://mender.s3-accelerate.amazonaws.com/temp_master/core-image-full-cmdline-qemux86-64.ext4" \
+  -o core-image-full-cmdline-qemux86-64.ext4 && \
+  curl -SL --fail "https://d1b0l86ne08fsf.cloudfront.net/mender-artifact/master/mender-artifact" \
+  -o /usr/bin/mender-artifact && \
+  curl -SL --fail "https://stress-client.s3-accelerate.amazonaws.com/release/mender-stress-test-client" \
+  -o downloaded-tools/mender-stress-test-client && \
```

Comment:
I don't think we need the mender-stress-test-client neither

Reply:
this is unfortunately required in one of the MenderAPI self checks, but as far as I can see it's just to check if there is an executable at that path...

Reply:
Ok. It makes little sense but let's not change MenderAPI in this task.